### ISSUE TITLE
Can't find SLInit on 10.0.10240.20708

### ISSUE THREAD

**Issue by loyejaotdiqr47123** [llccd/RDPWrapOffsetFinder#15]
(only nosymbol mode error
[termsrv_10.0.10240.20708.zip](https://github.com/user-attachments/files/16201716/termsrv_10.0.10240.20708.zip)
![image](https://github.com/user-attachments/assets/4d683aea-6fd9-4cdb-a50d-11141c959837)

**Comment by loyejaotdiqr47123** [comment]
@llccd

**Comment by loyejaotdiqr47123** [comment]
Try fix it.....
```
if (!*current && instruction.mnemonic == ZYDIS_MNEMONIC_MOV &&
                operands[0].type == ZYDIS_OPERAND_TYPE_MEMORY &&
                operands[0].mem.base == ZYDIS_REGISTER_RIP &&
                operands[0].mem.disp.size != 0 &&
                operands[1].type == ZYDIS_OPERAND_TYPE_REGISTER &&
                ((operands[1].reg.value == ZYDIS_REGISTER_EAX) || (operands[1].reg.value == ZYDIS_REGISTER_R14D)))
                *current = operands[0].mem.disp.value + IP - base;
 ```
 ```
bServerSku.x64=F13B8
ERROR: bRemoteConnAllowed not found
ERROR: bFUSEnabled not found
ERROR: bAppServerAllowed not found
ERROR: bMultimonAllowed not found
ERROR: lMaxUserSessions not found
ERROR: ulMaxDebugSessions not found
ERROR: bInitialized not found
```

**Comment by llccd** [comment]
fixed in e14adab
